cypress/integration/jsEmployeesTableDOM.spec.js

```diff
@@ -0,0 +1,244 @@
+'use strict';
+
+describe('Employees table', () => {
+  beforeEach('Open site', () => {
+    cy.visit('/');
+  });
+
+  it('should sort by name ASC', () => {
+    cy.get('th').contains('Name').click();
+
+    cy.get('tbody > :nth-child(1) > :nth-child(1)')
+      .contains('Airi Satou').should('exist');
```

Comment:
```suggestion
      .contains('Airi Satou');
```

отут should якраз зайвий